### DISCUSSION TITLE
Could not connect to Obsidian

### DISCUSSION THREAD

**Discussion by Kongulu** [coddingtonbear/obsidian-web#190] [Q&A]
Any suggestions for how to enable the connection with Obsidian?

![image](https://github.com/coddingtonbear/obsidian-web/assets/65778492/1fccf2eb-5546-44b6-be21-890051ed8fc5)

[obsidian-local-rest-api.bug_report.json](https://github.com/coddingtonbear/obsidian-web/files/15045645/obsidian-local-rest-api.bug_report.json)

**Comment by coddingtonbear** [comment]
Looking at the configuration, it does look like it has validated your API key; so you _should_ be good-to-go.  Are you really sure that Obsidian is running when you see that message?

**Reply by LightVolk** [reply]
Yes , but i think that bug can be reproduce when run extension without obsian and then start obsidian.

**Comment by Cmplx21** [comment]
Hi I am currently having the same issue. Obsidian is running with the Local REST API plugin enabled with default settings. I also downloaded the certificate that is listed in the plugins settings and added it to chrome's certs in the settings. In the obsidian-web plugin settings, I set my API key and see the icon that shows that the connection works. 
When I try to add a note for a webpage, I get the same error as @Kongulu though

**Reply by LightVolk** [reply]
May be this can help you - https://github.com/coddingtonbear/obsidian-web/discussions/188#discussioncomment-9397814

**Comment by qinbing8** [comment]
I encountered the same issue, which was resolved after I deleted the Obsidian Local REST API certificate and reinstalled the certificate correctly. For reference, please see the installation steps at: https://github.com/coddingtonbear/obsidian-web/wiki/How-do-I-get-my-browser-trust-my-Obsidian-Local-REST-API-certificate%3F#managing-previous-certificates

**Reply by Cmplx21** [reply]
Thanks a lot this actual solved it for me! I can't remember how I originally installed the certificate but just doing it as documented worked for me 😅
